web/portal/client/src/entities/Extension/Extension.tsx

```diff
@@ -127,6 +133,10 @@ const properties: ExtensionProperties = {
     label: _('Voicemail', { count: 1 }),
     required: true,
   },
+  locution: {
+    label: _('Voiceover and hang up'),
+    null: _('Direct hangup'),
```

Comment:
The field must be required, like in the other route types of Extension, so no null option must be available.